type-definitions/tests/immutable-flow.js

```diff
@@ -228,6 +228,9 @@ stringToNumber = Map({'a': 1}).removeIn([], 0)
 stringToNumber = Map({'a': 1}).mergeIn([], [])
 stringToNumber = Map({'a': 1}).mergeDeepIn([], [])
 
+anyMap = Map({'a': {}}).mergeIn(['a'], { b: 2 })
+anyMap = Map({'a': {}}).mergeDeepIn(['a'], { b: 2 })
```

Comment:
Needs some tests for false positives. Put `// $ExpectError` on a line preceding a code which should output an error for these.